src/api/middlewares/logRequest.ts

```diff
@@ -0,0 +1,21 @@
+import {
+    Request,
+    Response,
+} from "express";
+import { Container } from "typedi";
+import { Logger } from "winston";
+
+
+export default () => (req: Request, res: Response, next) => {
+    const logger: Logger = Container.get("logger");
+    let message = `Method: ${req.method}. Endpoint: ${req.path}.`;
+
+    const args = Object.entries(req.params);
+    if (args.length > 0) {
+        message += ` Args: ${args.map((entry: string[]) => entry.join(": ")).join(", ")}`;
+    }
+
+    logger.info(message);
```

Comment:
most logger tools has functionality of providing object to it. If you use it, you won't be needed to stringify onjects
For example:
```javascript
    logger.info('Request received', { method: req.method, url: req.url, params: req.params, body: req,body });
```

Reply:
Done.